src/main/java/org/EmailCarbonFootprint/EmailCarbonFootprintCalculator.java

```diff
@@ -1,59 +1,60 @@
 package org.EmailCarbonFootprint;
 
-import java.util.ArrayList;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
 public class EmailCarbonFootprintCalculator {
 
-	 private static final double CARBON_FOOTPRINT_OF_INBOX_EMAIL = 4.0;
-	 private static final double CARBON_FOOTPRINT_OF_SPAM_EMAIL = 0.3;
-	 private static final double CARBON_FOOTPRINT_OF_SENT_EMAIL = 4.9;
-	 static final int GRAM_TO_KILOGRAM = 1000;
-	 private static final String EMAIL_ID = "emailId";
-	 private static final String SOURCE = "source";
-	 private static final String INBOX = "inbox";
-	 private static final String SENT = "sent";
-	 private static final String SPAM = "spam";
-	 private String emailIds;
-	 private String source;
-	 private double inboxEmailsCount;
-	 private double sentEmailsCount;
-	 private double spamEmailsCount;
-	 static Map<String,String> emailCountDetails;
-	public EmailCarbonFootprintCalculator(String emailId, double inboxEmails,double sentEmails, double spamEmails) {
+	private static final double CARBON_FOOTPRINT_OF_INBOX_EMAIL = 4.0;
+	private static final double CARBON_FOOTPRINT_OF_SPAM_EMAIL = 0.3;
+	private static final double CARBON_FOOTPRINT_OF_SENT_EMAIL = 4.9;
+	static final int GRAM_TO_KILOGRAM = 1000;
+	private static final String EMAIL_ID = "emailId";
+	private static final String SOURCE = "source";
+	private static final String INBOX = "inbox";
+	private static final String SENT = "sent";
+	private static final String SPAM = "spam";
+	private String emailIds;
+	private String source;
+	private int inboxEmailsCount;
+	private int sentEmailsCount;
+	private int spamEmailsCount;
+	static Map<String, String> emailCountDetails;
+
+	public EmailCarbonFootprintCalculator(String emailId, int inboxEmails, int sentEmails, int spamEmails) {
```

Comment:
Function name "EmailCarbonFootprintCalculator" should be a verb like "calculateEmailCarvbonFootprint"
and also according to camel case

Reply:
This is constructer.